web/cpserver.go

```diff
@@ -148,6 +148,8 @@ func (sc *ServiceConfig) Serve(shutdown <-chan (interface{})) {
 		}
 
 		logger.Debug("Calling CC uiHandler")
+		w.Header().Add("Strict-Transport-Security","max-age=31536000")
```

Comment:
added requested changes, demo screen shot in the ticket https://jira.zenoss.com/browse/CC-2362